akka-docs/src/main/paradox/stream/operators/index-alphabetical.md

```diff
@@ -184,6 +186,7 @@
 * [fromIterator](source-or-flow/fromIterator.md)
 * [fromJavaStream](streamconverters/fromJavaStream.md)
 * [fromOutputStream](streamconverters/fromOutputStream.md)
+* [fromPath](fileio/fromPath.md)
```

Comment:
good :) what do you think about writing middle part in same case as the class; as they are indeed 1:1 to the classes?

Reply:
Kind of like scaladoc/javadoc do in urls